src/emulator/controller.ts

```diff
@@ -132,11 +132,36 @@ export async function startAll(options: any): Promise<void> {
       options.config.get("functions.source")
     );
 
+    let inspectFunctions = undefined;
```

Comment:
Does this make the compiler not too complainy?

```suggestion
    let inspectFunctions: number?;
```